src/maintainer/knowledge_base.rst

```diff
@@ -234,26 +234,32 @@ To skip building with a particular ``vc`` version, add a skip statement.
       build:
         - {{ compiler('cxx') }}
 
-Using vs2019
-^^^^^^^^^^^^^
+Using vs2022
+^^^^^^^^^^^^
 
-To use ``vs2019`` make the following changes:
+To use ``vs2022``, you'll need to edit ``conda_build_config.yaml`` and ``conda-forge.yml``.
 
-In ``conda_build_config.yaml`` file:
+In ``recipe/conda_build_config.yaml`` file:
 
 .. code-block:: yaml
 
     c_compiler:    # [win]
-    - vs2019       # [win]
+    - vs2022       # [win]
     cxx_compiler:  # [win]
-    - vs2019       # [win]
+    - vs2022       # [win]
 
+In ``conda-forge.yml`` file (at the root of the feedstock):
 
-For example see the changes made in the ``conda_build_config.yaml`` files in `this
-<https://github.com/conda-forge/libignition-msgs1-feedstock/pull/73/commits/81b5ee0e1d23f7f20427dd80d04cf1f7321b441d>`__ commit.
+.. code-block:: yaml
 
-After making these changes don't forget to rerender with ``conda-smithy`` (to rerender manually use ``conda smithy rerender`` from the command line).
+    azure:
+      settings_win:
+        pool:
+          vmImage: windows-2022
 
+You can look at the changes in `this PR <https://github.com/conda-forge/vcpkg-tool-feedstock/pull/41/files>`__.
```

Comment:
Since https://github.com/conda-forge/conda-smithy/pull/1720 got merged, this whole section can be removed (it's implied in the comments & cross-references from #1878, but perhaps it wasn't obvious).

Reply:
Ah. I saw the comments about the idea to change the default in `conda-smithy`, but did not see the linked and merged PR from Feb.

Reply:
21319730

I left the PR link in, but I added a comment to that PR to point out the section in `conda-forge.yml` can now be omitted.